src/main.ml

```diff
@@ -335,6 +335,18 @@ let cd_eject_cmd =
   Term.(ret (pure Xn.cd_eject $ common_options_t $ vbd)),
   Term.info "cd-eject" ~sdocs:_common_options ~doc ~man  
 
+let stat_vm_cmd =
+  let doc = "Stat a VM" in
```

Comment:
Documentation for the argument of the command is missing, please add `let vm = vm_arg "queried" in` or something similar.
(By the way, there are more commands that lack documentation of their arguments, but this is probably a different ticket.)